uportal-war/src/main/java/org/apereo/portal/shell/PortalShell.java

```diff
@@ -76,9 +76,6 @@ public static void main(String[] args) throws Exception {
                         new CompilerConfiguration(System.getProperties());
                 final GroovyShell shell = new GroovyShell(binding, conf);
                 shell.run(scriptFile, args);
-            } else {
```

Comment:
'Illegal class name "" in class file' from groovy.lang.GroovyShell.run(GroovyShell.java:517) call